src/main/preprocessing/preprocessing.py

```diff
@@ -1,103 +1,72 @@
 # Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
 
-import csv
+import os
 import logging
-from typing import List, Tuple, Optional
+from typing import List, Tuple
 
 import pandas as pd
 
 from src.main.util import consts
-from src.main.util.log_util import log_and_raise_error
-from src.main.preprocessing import activity_tracker_handler as ath
-from src.main.preprocessing.code_tracker_handler import handle_ct_file
-from src.main.preprocessing.activity_tracker_handler import handle_ati_file, get_ct_name_from_ati_data, \
-    get_files_from_ati
-from src.main.util.file_util import get_original_file_name, get_all_file_system_items, data_subdirs_condition, \
-    get_parent_folder_name, get_output_directory, write_result, extension_file_condition
+from src.main.util.consts import EXTENSION, CODE_TRACKER_COLUMN, TEST_MODE, ACTIVITY_TRACKER_COLUMN
+from src.main.util.file_util import get_output_directory, get_all_file_system_items, all_items_condition, \
+    extension_file_condition, get_file_with_max_size, create_directory, get_name_from_path, create_file, \
+    get_content_from_file
 
 log = logging.getLogger(consts.LOGGER_NAME)
 
 
-def is_ct_file(csv_file: str, column: consts.CODE_TRACKER_COLUMN = consts.CODE_TRACKER_COLUMN.CHOSEN_TASK) -> bool:
-    with open(csv_file, encoding=consts.ISO_ENCODING) as f:
-        reader = csv.reader(f)
-        try:
-            if column.value in next(reader):
-                return True
-        except StopIteration:
-            return False
-    return False
-
-
-def __get_real_ati_file_index(files: List[str]) -> int:
-    count_ati = 0
-    ati_index = -1
-    for i, f in enumerate(files):
-        if consts.ACTIVITY_TRACKER_FILE_NAME in f and not is_ct_file(f):
-            count_ati += 1
-            ati_index = i
-            if count_ati >= 2:
-                log_and_raise_error('The number of activity tracker files is more than 1', log)
-    return ati_index
-
-
-def __has_files_with_same_names(files: List[str]) -> bool:
-    original_files = list(map(get_original_file_name, files))
-    return len(original_files) != len(set(original_files))
+def __partition_into_ct_and_ati_files(files: List[str]) -> Tuple[List[str], List[str]]:
+    ati_files = [f for f in files if 'ide-events' in f]
```

Comment:
let's use a variable ACTIVITY_TRACKER_FILE_NAME instead of 'ide-events' (we already have it in consts)